ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -157,6 +157,41 @@ class AnalysisSectionDamages(AnalysisSectionBase):
     file_name: Optional[Path] = None
 
 
+@dataclass
+class AnalysisSectionAdaptation(AnalysisSectionBase):
+    """
+    Reflects all possible settings that an adaptation analysis section might contain.
+    """
+
+    analysis: str = ""
+    discount_rate: float = 0.0
+    time_horizon: float = 0.0
```

Comment:
After discussion with Chamidu, `time_horizon` should be an argument of `AnalysisSectionAdaptationOption`